src/mergeHead.ts

```diff
@@ -0,0 +1,64 @@
+import XXH from 'xxhashjs';
+
+interface Options {
+  space?: number | string;
+}
+
+const stringifyLD = (options: Options) => function () {
+  const jsonLd = this.$options.jsonld.call(this);
+
+  if (jsonLd !== null) {
+    const stringifiedJson = JSON.stringify(jsonLd, null, options.space);
+    const innerHTML = options.space === 0 ? stringifiedJson : `\n${stringifiedJson}\n`;
+
+    const hash = XXH.h32(innerHTML, 0).toString(16);
+    const hid = `nuxt-jsonld-${hash}`;
+
+    return {
+      script: [
+        {
+          hid,
+          type: 'application/ld+json',
+          innerHTML,
```

Comment:
As of vue-meta v2.2 you can set a json prop directly

Reply:
True, you cant change that at the moment

Reply:
Will leave this as is, but i don't actually see the utility of having the `space` option at all, since in production head code gets minified most times.

Reply:
Will leave this as is, but i don't actually see the utility of having the `space` option at all, since in production head code gets minified most times.